java/code_to_gast/java_expression.py

```diff
@@ -53,6 +54,25 @@ def list_to_attribute_value_node(object_list):
 
 def class_declaration_to_gast(node):
     gast = {"type": "root"}
-    # only support for one function currently
-    gast["body"] = java_router.node_to_gast(node.body[0])
+    '''
+    If the method in class is main function only translate that method and ignore function header
+    We are assuming user only wants body of function translated
+    '''
+    if type(node.body[0]) == javalang.tree.MethodDeclaration and node.body[
+            0].name == "artifical_wrapper_WkVHC":
```

Comment:
What does the "artificial_wrapper_WkVHC" mean?

Reply:
Maybe add a comment to explain

Reply:
Ill add a comment - this is a function name that signifies that the user just typed in commands ie System.out.println(). We need to store the user's input in a class and function to compile in java. We don't store this function as a function declaration in the gast we just store the body of this function so only the commands are translated.

Reply:
Pull this into a top level constant and add a comment there.